src/oss-health/HealthTool.cs

```diff
@@ -29,7 +29,7 @@ public HealthTool() : this(new DefaultHttpClientFactory())
 
             if (packageManager != null)
             {
-                string? content = await packageManager.GetMetadata(purl, false);
+                string? content = await packageManager.GetMetadata(purl, useCache: false);
```

Comment:
This shouldn't be hard coded false. Might need to introduce a parameter.

Reply:
I can just set it as the default, don't need to mess with the health tool or anything.

Reply:
I think either keep the previous behavior of using cache or add a parameter. 

Up to you if you want to explicitly specify use cache or let it be implicit if you go with the first option.